### ISSUE TITLE
🔎 Improving user research experience

### ISSUE THREAD

**Issue by 0xEzSwim** [real-token/realtoken-yam-interface#9]
In order to optimize the search for properties on YAM (= avoid going through 3 different websites) I suggest a finer filtering system like Notion, Jira or Click-up:
![image](https://user-images.githubusercontent.com/44302768/222884451-d112996f-c24b-4a08-8166-b1e853275f1e.png)

Each criteria is a column of the table on which we can filter with operators appropriate to the type of the column: 
- Boolean & Multiselect: equal, not equal
- String: contains, does not contain, begins with, ends with, is empty, is not empty
- Number & Date: equals, not equals, is null, is not null, greater than, greater than or equals, less than, less than or equals

The URL would also contain the base64 filter so that one could use the "previous" button of the browser to go back to the previous filter and the community could share their search results.

In addition to this filter a summary in button/tag format to quickly understand what criteria the user is filtering on is welcome:
![image](https://user-images.githubusercontent.com/44302768/222885240-457d639c-6f6e-42d7-ae08-d91768b64b87.png)

This feature could be used to save filters, for example "Under valued" (priceDiff < 0) or "Holdings" (name contains "holding") which would allow the user to observe only the sub-markets that interest him.

In summary this filter has the advantage of promoting sharing and improving properties research.
